CHANGELOG.md

```diff
@@ -9,6 +9,43 @@
 
 **Changed**:
 
+- **decidim-core**: Introduce coauthorable concern and coauthorship model. [\#3310](https://github.com/decidim/decidim/pull/3310)
+- **decidim-core**: Force user_group.name uniqueness in user_group test factory. (https://github.com/decidim/decidim/pull/3290)
```

Comment:
This changelog seems wrong

Reply:
Sorry @mrcasals you mean the first at line 12 or the second at line 13? I guess it is the second that I have added by error correct?